### ISSUE TITLE
Please include all the financial information for NSE (Indian stocks ) too

### ISSUE THREAD

**Issue by kumarrahulicfai** [ranaroussi/yfinance#311]
I am not able to fetch the Indian stocks financial data (fiancials, earning, cashflow etc). Please include them too.

**Comment by ZaphodBeeblebrox12** [comment]
I am going to post it here since my question is similar to @kumarrahulicfai This is a NSE security. 
I tried the following code:
```
import yfinance as yf
stock = 'ACC.NS'
pg = yf.Ticker(stock)

print("Short Name: " + pg.info['shortName'])
print("Long Name: " + pg.info['longName'])
```
It is not fetching even the name of security.  Is there a work around? I am getting an error:
`IndexError: list index out of range
`

**Comment by rodrigobercini** [comment]
Hey there, try applying the fixes described in PR #179.

Just be aware that that fix returns inconsistent data for non-US tickers.
